### ISSUE TITLE
santa extension error message

### ISSUE THREAD

**Issue by tonynotears** [trailofbits/osquery-extensions#39]
after executing 
`sudo osqueryi --extension /var/osquery/extensions/trailofbits_osquery_extensions.ext --allow_unsafe=true`

getting the following error:

```
osquery> SELECT * FROM santa_denied;
Error: no such table: santa_denied
osquery> SELECT * FROM santa_allowed;
Error: no such table: santa_allowed
```

but seems like the following worked:
`SELECT * FROM santa_rules;`

Please provide some guidance on how to troubleshoot. Appreciate it.

**Comment by tonynotears** [comment]
@mike-myers-tob I grabbed it from your most recent release.

**Comment by mike-myers-tob** [comment]
> @mike-myers-tob I grabbed it from your most recent release. https://github.com/trailofbits/osquery-extensions/releases/download/1.1/trailofbits_osquery_extensions-macos-v1.1.zip

I think that's the reason; we need to put up a newer release soon. In the meantime, if you build from source, it will include the new Santa tables.

**Comment by tonynotears** [comment]
also getting compile error with CMakeLists.txt
CMakeLists.txt:15: *** missing separator.  Stop.

Whitespace/Tab issue?

**Comment by alessandrogario** [comment]
Hello @tonynotears 
It seems like osquery is being compiled with SKIP_TESTS set to 1 (environment variable); if that's not the case, then the dependencies are missing. Make sure the system has been initialized correctly:

```
make sysprep
make deps
```

**Comment by tonynotears** [comment]
@alessandrogario  this is the message compling in the folder:
$ make sysprep
make: *** No rule to make target `sysprep'.  Stop.

Tried both in the /santa folder and in the /osquery-extention folder

**Comment by alessandrogario** [comment]
Hey @tonynotears 

The build instructions are located in the README file: https://github.com/trailofbits/osquery-extensions#building

It is similar to how osquery is normally built, except you need to create a symbolic link to our source folder.

**Comment by tonynotears** [comment]
now getting this error 
```
TypeError: cannot use a string pattern on a bytes-like object
-- Unable to detect osquery platform: ./tools/get_platform.py failed
```

Maybe a osquery binary issue? will post that in the #general channel in osquery

**Comment by alessandrogario** [comment]
This has been fixed with the newly release 1.2 version (https://github.com/trailofbits/osquery-extensions/commit/724123ac19ecd801de379cdfac3b939ca4011001)
